### DISCUSSION TITLE
mjx rc car protocol？

### DISCUSSION THREAD

**Discussion by nzjott** [pascallanger/DIY-Multiprotocol-TX-Module#915] [General]
hi just got my radiomaster tx12 here and want to use it on mjx 14302 hyper go rc cars, but I can't find a right protocol for it. I attached original rc for it.
![20231214_234802](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/68474235/a90b8504-b522-41c4-98a3-ea560a3f4d25)
![20231214_234749](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/68474235/a5e4187c-9429-48f3-bb02-5a1313740c77)

**Comment by pascallanger** [comment]
The main IC does not have a proper marking so I don't know which RF component is being used. I need access to the TX and RX to reverse the protocol, if you are willing to ship the hw to France PM me.

**Reply by nzjott** [reply]
It's really a pain to ship thing out from nz, is there anyway I can dump the data and post ?  I also have a radiolink r4eh rx and rc4g tx and want to see if there is anyway to dump things out of them

**Comment by pascallanger** [comment]
For the mjx, you can always try to do a XN297Dump in auto mode.

**Comment by pascallanger** [comment]
For the radiolink receiver have you tested the existing protocol : https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#radiolink---74

**Reply by nzjott** [reply]
I tried but seems not work. As the old radiolink rc4g tx and r4eh-h / r4eh-g aren't compatible with the newer rc6gs radios. They even have free receiver exchange from r4eh to r6fg when the rc6gs first came out. ..

**Comment by MaxxerMax** [comment]
Hello, I am interested in the protocol for MJX cars as well. Are there any news? Thanks!

**Reply by pascallanger** [reply]
Unless I have access to a TX + RX nothing will happen.

**Reply by MaxxerMax** [reply]
I´ve checked the transfer costs. Approx 40€ in total for both ways. If anyone else would participate in a cost share I could send you the equipment. Lets see if more are interested in this. :-) 

Is there otherwise a possiblity to dump the protocol with a pc and a modified wifi card?

**Reply by CGPro856** [reply]
Has this protocol been added yet? If not, it looks like I could buy a T3B/R3B set on AliExpress for about $25 and just have it delivered to Pascal. What are the odds this would work?

**Reply by pascallanger** [reply]
There is no way to say that the protocol can be reversed... But I'm always trying to my best to get it done. Let me know if you want to give it a try.

**Reply by CGPro856** [reply]
Let’s give it a shot. I sent you an email.

**Reply by pascallanger** [reply]
I haven't received an email (nothing in mail or spam)... Have you used my yahoo email address? or another way?

**Reply by CGPro856** [reply]
I think it was just a reply to this thread that I thought was going directly to you. Send me an email at CGPro856@gmail.com.

**Reply by Nemuritor01** [reply]
I'm also interested in the mjx protocol. Did you manage to ship a tx and rx? It's 15 bucks on Ali, so I could also help out to ship one to Pascal if necessary.

**Reply by CGPro856** [reply]
I did ship one and he has received it.

**Reply by Nemuritor01** [reply]
Great. Keep my fingers crossed, Pascal is successful.

**Reply by peter--s** [reply]
Hi, what happened to this course of actions?
https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/990 seems related, both look stale :-(

**Reply by CGPro856** [reply]
Pascal said he received my shipment and I have not heard from him since. All my emails have gone unanswered. At this point I’ve given up.

**Reply by peter--s** [reply]
Thanks for your effort and your feedback!

**Reply by IKKI-FPV** [reply]
Nothing???

**Reply by CGPro856** [reply]
Not a peep since he told me that he had received the transmitter and receiver that I sent to him…

**Comment by rwebb616** [comment]
I am also interested - I have a TX and RX .. not sure how much it would cost to ship from the US.. How does one go about reverse engineering it?  This seems to be an autobind protocol and it's not compatible with any of the mjxq entries already in EdgeTX.  The receiver I have is only marked with R30 .. my car is a hypergo H16BM brushless.  I put an R85C that came with my MT12 in it but I think it would be cool to get the protocol working natively.

**Comment by c0ldtech** [comment]
Why not just swap out the receiver for a DumboRC one and get a DumboRC remote? That was one of the first things I did for my MJX cars. The receivers are very cheap, and the transmitters don't cost much, either (and are way better than the ones that come with the MJX cars).

**Comment by peter--s** [comment]
I just found https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/1124 (which does link here, but not vice-versa, I think)
